manifests/init.pp

```diff
@@ -81,7 +81,11 @@
 #
 # $puppet_group::               Groups of Foreman proxy user
 #
-# $autosignfile::               Path to the autosign file
+# $puppetca_modular::           Using the current modular PuppetCa implementation?
```

Comment:
How about: Whether the PuppetCA implementation is modular. This is true for 1.19 or later.

Reply:
Oh, and I think all of these should be under advanced parameters. I think we don't want to expose these options to every user via `--help` and for 1.19 we should ensure proper defaults and migrations.